README.md

```diff
@@ -83,15 +85,19 @@ Extensions that use single-phase initialization need to call
 [`PyUnstable_Module_SetGIL()`](https://docs.python.org/3.13/c-api/module.html#c.PyUnstable_Module_SetGIL)
 in the module's initialization function:
 
-```
-PyObject *mod = PyModule_Create(&module);
-if (mod == NULL) {
-    return NULL;
-}
+```c
+PyMODINIT_FUNC
+PyInit__module(void)
+{
+    PyObject *mod = PyModule_Create(&module);
+    if (mod == NULL) {
+        return NULL;
+    }
 
 #ifdef Py_GIL_DISABLED
-PyModule_ExperimentalSetGIL(mod, Py_MOD_GIL_NOT_USED);
+    PyModule_ExperimentalSetGIL(mod, Py_MOD_GIL_NOT_USED);
```

Comment:
Looks like this should be updated to `PyUnstable_Module_SetGIL`

Reply:
Right. Thanks @rgommers!